codalab/client/local_bundle_client.py

```diff
@@ -928,7 +943,8 @@ def update_worksheet_items(self, worksheet_info, new_items):
         last_item_id = worksheet_info['last_item_id']
         length = len(worksheet_info['items'])
         worksheet = self.model.get_worksheet(worksheet_uuid, fetch_items=False)
-        check_worksheet_has_all_permission(self.model, self._current_user(), worksheet)
+        if not spec_util.is_dashboard(worksheet_info['name']):
```

Comment:
Huh.. is there a reason we wanna do this rather than making it part of the configuration process? I felt like counting on the admin being the first to visit the dashboard is not super robust.